modules/nf-core/biscuit/bsconv/meta.yml

```diff
@@ -1,52 +1,48 @@
-name: biscuit_bsconv
-description: Summarize and/or filter reads based on bisulfite conversion rate
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "biscuit_bsconv"
+description: write your description here
 keywords:
-  - biscuit
-  - DNA methylation
-  - WGBS
-  - scWGBS
-  - bisulfite sequencing
-  - aligner
-  - bam
-  - filter
+  - sort
+  - example
+  - genomics
 tools:
-  - biscuit:
+  - "biscuit":
       description: A utility for analyzing sodium bisulfite conversion-based DNA methylation/modification data
       homepage: https://huishenlab.github.io/biscuit/
-      documentation: https://huishenlab.github.io/biscuit/docs/subcommand_help.html#biscuit-bsconv
+      documentation: https://huishenlab.github.io/biscuit/biscuitblaster/
```

Comment:
link doesn't work for me - perhaps you meant https://huishenlab.github.io/biscuit/biscuitsifter/ ?